src/sas/qtgui/Calculators/UI/SldPanel.ui

```diff
@@ -338,6 +338,9 @@
         <property name="text">
          <string>Recalculate</string>
```

Comment:
SLD Calculator Tool documentation says the button caption is "Calculate". Either the code or the documentation should be changed.